File: PlayMinds-Back/src/main/java/com/coderhouse/pmb/Entitys/Comment.java
package com.coderhouse.pmb.Entitys;
import jakarta.persistence.*;
import lombok.Data;

import java.io.Serializable;
import java.sql.Date;


@Data
@Entity
@Table(name = "comments")
public class Comment implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id_comment;
    private String game_id_game;
    private String user_email;
    private String comment;
    private Date date_comment;
}

File: ExamplesOBJ/models/Entitys/Comment.ts
export class Comment {
  game_id_game: string;
  user_email: string;
  comment: string;
  date: Date;

  constructor(
    game_id_game: string,
    user_email: string,
    comment: string,
    date: Date
  ) {
    this.game_id_game = game_id_game;
    this.user_email = user_email;
    this.comment = comment;
    this.date = date;
  }
}
